src/RamProgram.h

```diff
@@ -118,13 +118,27 @@ class RamProgram : public RamNode {
 
     /** Create clone */
     RamProgram* clone() const override {
+        typedef std::map<const RamRelation*, const RamRelation*> RefMapType;
```

Comment:
Why the 'type' in RefMapType?

Reply:
indeed, not needed anymore; before I had a RamMapper class and needed to pass arguments having a replicated use of the type. However, with makeLambdaRamMapper the current environment can be used.